apps/server/src/services/window.ts

```diff
@@ -257,7 +257,7 @@ async function configureWebContents(webContents: WebContents, spellcheckEnabled:
 }
 
 function getIcon() {
-    return path.join(RESOURCE_DIR, "images/app-icons/png/256x256" + (isDev ? "-dev" : "") + ".png");
+    return path.join(RESOURCE_DIR, "../public/assets/icon.png");
```

Comment:
^ The previous path doesn't exist in the (current) release build.
The new path does exist in the flake build (and the release `app.asar`), but it doesn't seem to load, unfortunately..